src/main/java/com/solr/clientwrapper/rest/SolrSchemaResource.java

```diff
@@ -1,104 +1,104 @@
-		this.createSolrSchema = createSolrSchema;
-		this.deleteSolrSchema = deleteSolrSchema;
-		this.updateSolrSchema = updateSolrSchema;
-		this.getSolarSchema = getSolarSchema;
-	}			
-
-	@PostMapping("/create")
-	@Operation(summary = "/create-schema", security = @SecurityRequirement(name = "bearerAuth"))
-	public ResponseEntity<SolrSchemaResponseDTO> create(
-			@RequestBody SolrSchemaDTO newSolrSchemaDTO) {
-		log.debug("Solr Schema Create");
-		log.debug("Received Schema as in Request Body: {}", newSolrSchemaDTO);
-		SolrSchemaResponseDTO solrResponseDTO = 
-				createSolrSchema.create(
-						newSolrSchemaDTO.getTableName(), 
-						newSolrSchemaDTO.getName(), 
-						newSolrSchemaDTO);
-		if(solrResponseDTO.getStatusCode() == 200)
-			return ResponseEntity.status(HttpStatus.OK).body(solrResponseDTO);
-		else
-			return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(solrResponseDTO);
-	}
-
-	@DeleteMapping("/delete/{tableName}/{name}")
-	@Operation(summary = "/delete-schema", security = @SecurityRequirement(name = "bearerAuth"))
-	public ResponseEntity<SolrSchemaResponseDTO> delete(
-			@PathVariable String tableName, 
-			@PathVariable String name) {
-		log.debug("Schema Delete");
-		SolrSchemaResponseDTO solrSchemaResponseDTO = deleteSolrSchema.delete(tableName, name);
-		if(solrSchemaResponseDTO.getStatusCode() == 200)
-			return ResponseEntity.ok().body(solrSchemaResponseDTO);
-		else
-			return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(solrSchemaResponseDTO);
-		
-	}
-
-	@PutMapping("/update/{tableName}/{name}")
-	@Operation(summary = "/update-schema", security = @SecurityRequirement(name = "bearerAuth"))
-	public ResponseEntity<SolrSchemaResponseDTO> update(
-			@PathVariable String tableName, 
-			@PathVariable String name, 
-			@RequestBody SolrSchemaDTO newSolrSchemaDTO) {
-		log.debug("Solr schema update");
-		log.debug("Received Schema as in Request Body: {}", newSolrSchemaDTO);
-		SolrSchemaResponseDTO solrSchemaDTO = updateSolrSchema.update(tableName, name, newSolrSchemaDTO);
-		SolrSchemaResponseDTO solrResponseDTO = new SolrSchemaResponseDTO(solrSchemaDTO);
-		if(solrResponseDTO.getStatusCode() == 200)
-			return ResponseEntity.status(HttpStatus.OK).body(solrResponseDTO);
-		else
-			return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(solrResponseDTO);
-	}
-
-	@GetMapping("/get/{tableName}/{name}")
-	@Operation(summary = "/get-schema", security = @SecurityRequirement(name = "bearerAuth"))
-	public ResponseEntity<SolrSchemaResponseDTO> get(
-			@PathVariable String tableName, 
-			@PathVariable String name) {
-		log.debug("get solar schema");
-		SolrSchemaResponseDTO solrResponseDTO = getSolarSchema.get(tableName, name);
-		if(solrResponseDTO.getStatusCode() == 200)
-			return ResponseEntity.status(HttpStatus.OK).body(solrResponseDTO);
-		else
-			return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(solrResponseDTO);
-	}
-}
+//package com.solr.clientwrapper.rest;
```

Comment:
Uncomment this file and test again if it passes all the Unit and Integration tests.
And _avoid interfering with already created layers without consulting the concerned developers;_ **_this might create huge conflict at times otherwise._**